benchmarks/programs/test_kernel/macro_output.txt

```diff
@@ -0,0 +1,115 @@
+fn function_name(var_n : usize) -> usize
```

Comment:
why not use u32 instead of usize? u32 is more consistent and has no ambiguity on any platform